swift/README.md

```diff
@@ -1,36 +1,43 @@
-# AWS SDK for Swift code examples 
+# AWS SDK for Swift code examples
+## Overview
+The code examples in this topic show you how to use the AWS SDK for Swift with AWS.
 
-## Purpose
+The AWS SDK for Swift provides a Swift API for AWS infrastructure services. Using the SDK, you can build applications on top of Amazon S3, Amazon EC2, Amazon DynamoDB, and more.
 
-These examples demonstrate how to use the AWS SDK for Swift to access and use
-Amazon Web Services from your projects.
+## Types of code examples
+* **Single-service actions** - Code examples that show you how to call individual service functions.
 
-## Prerequisites
+* **Single-service scenarios** - Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
 
-You must have an AWS account, and have configured your default credentials and
-AWS Region as described in the [Getting started
-section](https://docs.aws.amazon.com/sdk-for-swift/latest/developer-guide/getting-started.html)
-of the _AWS SDK for Swift Developer Guide_.
+* **Cross-service examples** - Sample applications that work across multiple AWS services.
 
-## Running the code
+Single-service actions and scenarios are organized by AWS service in the
```

Comment:
The template has changed slightly to include a `### Finding code examples` heading above these navigational sentences.

Reply:
Added.